django/core/mail/__init__.py

```diff
@@ -145,13 +165,28 @@ def mail_managers(
     """Send a message to the managers, as defined by the MANAGERS setting."""
     if not settings.MANAGERS:
         return
-    if not all(isinstance(a, (list, tuple)) and len(a) == 2 for a in settings.MANAGERS):
-        raise ValueError("The MANAGERS setting must be a list of 2-tuples.")
+    recipients = settings.MANAGERS
+
+    # RemovedInDjango70Warning.
+    if all(isinstance(a, (list, tuple)) and len(a) == 2 for a in settings.MANAGERS):
+        warnings.warn(
+            "Using (name, address) pairs in the MANAGERS setting is deprecated."
+            " Replace with a list of email address strings.",
+            RemovedInDjango70Warning,
+        )
+        recipients = [a[1] for a in recipients]
+
+    if not isinstance(recipients, (list, tuple)) or not all(
+        isinstance(address, (str, Promise)) for address in recipients
+    ):
+        raise ImproperlyConfigured(
+            "The MANAGERS setting must be a list of email address strings."
+        )
```

Comment:
There's quite a bit of near-duplicate code here from mail_admins(). But that was true before this change, too. I didn't feel it was in-scope (or even necessarily the right idea) to try to share the duplicate code, but I could pretty easily be convinced otherwise.

Reply:
I won't comment on if it's necessary or even the best way in this situation, but what is often done in these situations is to make another PR that only cleans up the duplication, and then rebase this PR once the cleanup is merged.

Reply:
`def _mail_important_people(recipients, subject, message, ...)` ? Seems quite bespoke but it _is_ the same stuff over a couple lines modulo the `recipients`....

Reply:
OK, I'm convinced. Extracted the common code into a shared `_send_server_message()`, as a separate commit prior to the ticketed change.